Common subdirectories: oci-cli-3.18.1.orig/.github and oci-cli-3.18.1/.github
diff -u oci-cli-3.18.1.orig/requirements.txt oci-cli-3.18.1/requirements.txt
--- oci-cli-3.18.1.orig/requirements.txt	2022-10-04 22:54:50.000000000 +0200
+++ oci-cli-3.18.1/requirements.txt	2022-10-10 14:22:35.258579182 +0200
@@ -2,41 +2,41 @@
 # (https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format),
 # you may need to use the --extra-index-url option instead.
 
-appdirs==1.4.3
+appdirs>=1.4.3
 arrow>=1.0.0
 certifi
 cffi>=1.9.1
-click==7.1.2
-coverage==4.5.2
-cryptography>=3.2.1,<=37.0.2
-httpsig-cffi==15.0.0
-Jinja2==3.0.3
-jmespath==0.10.0
-ndg-httpsclient==0.4.2
-mock==2.0.0
-oci==2.85.0
-packaging==20.2
-pluggy==0.13.0
-py==1.10.0
-pyasn1==0.2.3
-pyOpenSSL>=17.5.0,<=22.0.0
-pycparser==2.20
-pyparsing==2.2.0
-pytest==3.2.3
-pytest-cov==2.5.1
-python-dateutil>=2.5.3,<3.0.0
+click>=7.1.2
+coverage>=4.5.2
+cryptography>=3.2.1
+httpsig-cffi>=15.0.0
+Jinja2>=3.0.3
+jmespath>=0.10.0
+ndg-httpsclient>=0.4.2
+mock>=2.0.0
+oci>=2.85.0
+packaging>=20.2
+pluggy>=0.13.0
+py>=1.10.0
+pyasn1>=0.2.3
+pyOpenSSL>=17.5.0
+pycparser>=2.20
+pyparsing>=2.2.0
+pytest>=3.2.3
+pytest-cov>=2.5.1
+python-dateutil>=2.5.3
 pytz>=2016.10
-requests==2.21.0
+requests>=2.21.0
 setuptools<45
 six>=1.15.0
-sphinx==3.3.0
-sphinx-rtd-theme==0.4.3
-terminaltables==3.1.0
-tox==3.23.0
-vcrpy==1.13.0
-virtualenv==20.13.0
-pytest-xdist==1.22.2
-pytest-forked==1.0.2
-PyYAML>=5.4,<6
-docutils==0.15.2
-prompt-toolkit==3.0.29
+sphinx>=3.3.0
+sphinx-rtd-theme>=0.4.3
+terminaltables>=3.1.0
+tox>=3.23.0
+vcrpy>=1.13.0
+virtualenv>=20.13.0
+pytest-xdist>=1.22.2
+pytest-forked>=1.0.2
+PyYAML>=5.4
+docutils>=0.15.2
+prompt-toolkit>=3.0.29
Common subdirectories: oci-cli-3.18.1.orig/scripts and oci-cli-3.18.1/scripts
Common subdirectories: oci-cli-3.18.1.orig/services and oci-cli-3.18.1/services
diff -u oci-cli-3.18.1.orig/setup.py oci-cli-3.18.1/setup.py
--- oci-cli-3.18.1.orig/setup.py	2022-10-04 22:54:50.000000000 +0200
+++ oci-cli-3.18.1/setup.py	2022-10-10 14:21:56.146255580 +0200
@@ -29,23 +29,23 @@
     readme = f.read()
 
 requires = [
-    'oci==2.85.0',
+    'oci>=2.85.0',
     'arrow>=1.0.0',
     'certifi',
-    'click==7.1.2',
-    'cryptography>=3.2.1,<=37.0.2',
-    'jmespath==0.10.0',
-    'python-dateutil>=2.5.3,<3.0.0',
+    'click>=7.1.2',
+    'cryptography>=3.2.1',
+    'jmespath>=0.10.0',
+    'python-dateutil>=2.5.3',
     'pytz>=2016.10',
     'six>=1.15.0',
-    'terminaltables==3.1.0',
-    'pyOpenSSL>=17.5.0,<=22.0.0',
-    'PyYAML>=5.4,<6',
-    'prompt-toolkit==3.0.29'
+    'terminaltables>=3.1.0',
+    'pyOpenSSL>=17.5.0',
+    'PyYAML>=5.4',
+    'prompt-toolkit>=3.0.29'
 ]
 
 extras = {
-    'db': ['cx_Oracle==8.3']
+    'db': ['cx_Oracle>=8.3']
 }
 
 fips_libcrypto_file = os.getenv("OCI_CLI_FIPS_LIBCRYPTO_FILE")
Common subdirectories: oci-cli-3.18.1.orig/src and oci-cli-3.18.1/src
Common subdirectories: oci-cli-3.18.1.orig/tests and oci-cli-3.18.1/tests
